src/Telemetry/AzureDevOpsTimeline/AzureDevOpsTimelineOptions.cs

```diff
@@ -15,5 +15,6 @@ public class AzureDevOpsTimelineOptions
         public string Interval { get; set; }
         public string ParallelRequests { get; set; }
         public string BuildBatchSize { get; set; }
+        public string LogScrapingTimeout { get; set; }
```

Comment:
What happens if you make this a `TimeSpan` right away? And change to `00:00:00` won't that work? (asking)

Reply:
We'd have to parse it in a different place (Program.cs) where we're setting properties for the AzureDevOpsTimelineOptions

Reply:
The options framework should handle parsing his automatically if we are calling "Bind". Which is what I assume we do (since that's what we need to do to make secret-reloading work)

Reply:
We are not using Bind for AzureDevOpsTimelineOptions, we handle things like "InitialDelay" the same way